testcases/crypto/aes_func.c

```diff
@@ -69,6 +69,11 @@ CK_RV do_EncryptDecryptAES(struct generated_test_suite_info *tsuite)
         goto testcase_cleanup;
     }
 
+    /* Skip tests if pkey = false, but the slot doesn't support CKM_AES_XTS*/
```

Comment:
Style: Add a blank please before the closing `*/`
Same for all places further down.